jwt/types.py

```diff
@@ -1,3 +1,5 @@
-from typing import Any, Dict
+from typing import Any, Callable, Dict
 
 JWKDict = Dict[str, Any]
+
+HashlibHash = Callable[..., Any]
```

Comment:
Inspired by https://github.com/python/typeshed/blob/main/stdlib/hmac.pyi#L10 (may be narrowed to a more specific type in the future). Note that the type alias from typeshed is incorrect (`Callable[[], Any]` means that the callable can't take any argument, which isn't the case).